doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2731,16 +2731,18 @@ Performing this procedure destroys any existing data and cannot be undone. Back
 
 To configure database replication, you must configure one {product-title} instance to act as a global copy, and one or more other instances to act as remote copies. This database replication relationship can only be configured on {product-title} instances that are of the same version. 
 
-You can configure centralized administration via the database replication configuration steps described in this section. Once configured, {product-title} supports life cycle management operations that can be initiated from the global copy and processed and executed on the remote copy. 
-
 [NOTE]
-======
-You must configure at least one remote copy before you can configure the global copy. Changes to the role of a {product-title} instance take several minutes to take effect.
-======
+====
+When you configure database replication in this version of {product-title_short}, centralized administration gets automatically enabled and no further configuration is needed.
```

Comment:
Here again I would restructure this sentence to remove the passive voice to something like "Configuring database replication in this version of <product title> automatically enables...